numeral.js

```diff
@@ -394,8 +411,7 @@
             if (format.indexOf('.') === 0) {
                 w = '';
             }
-
-            return ((negP && neg) ? '(' : '') + ((!negP && neg) ? '-' : '') + ((!neg && signed) ? '+' : '') + w + d + ((ord) ? ord : '') + ((abbr) ? abbr : '') + ((bytes) ? bytes : '') + ((negP && neg) ? ')' : '');
+            return ((negP && neg) ? '(' : '') + ((!negP && neg) ? '-' : '') + ((!neg && signed) ? '+' : '') + w + d + ((ord) ? ord : '') + ((abbr && !omitAbbr) ? abbr : '') + ((bytes && !omitAbbr) ? bytes : '') + ((negP && neg) ? ')' : '');
```

Comment:
Yeah, what @metathinker said. I generally want to make less change to the forked project if possible. 

But I agree that the current code is completely illegible.